paddle/fluid/platform/gpu_info.cc

```diff
@@ -64,6 +74,16 @@ int GetCUDADriverVersion(int id) {
   return driver_version;
 }
 
+bool TensorCoreAvailable() {
+#if CUDA_VERSION >= 9000
+  int device = GetCurrentDeviceId();
+  int driver_version = GetCUDAComputeCapability(device);
```

Comment:
maybe here we can give a fatal error, when it's driver version <= 70 but CUDA_VERSION >= 9000

Reply:
I will add a `WARNING` in next PR.